docs/advanced/arrow_function.md

```diff
@@ -0,0 +1,113 @@
+## Arrow Functions
+Arrow functions are functions defined using the arrow (=>) symbol, and they are commonly used to create anonymous functions or to pass as callback functions to other functions.
+### Supported Patterns
+
+```yaml
+name: Advanced Arrow Functions
+```
+
+#### Semantic: Arrow Functions
+
+##### Examples
+
+###### Call + Chained call
```

Comment:
Refactor all test cases' names: Do not simply copy and paste PyCG's name, write its own name using JS terms.

Also to mark the relationship between PyCG's dataset and our migration, please add **comment block** for each test case if it is migrated from PyCG, the format is `<group name>/<case name>`:

```markdown
<!--args/assigned_call-->
```

if the test case is one-to-one;

```markdown
<!--args/assigned_call,args/call-->
```

if our test case combines PyCG's two.

Reply:
If the test case is created on our own, not migrated from PyCG, this comment block can be omitted.

Reply:
Update: Please add space in the comment block.

You can refer to https://github.com/xjtu-enre/ENRE-ts/blob/4bad5348548118faef30cf6eaa2bf76872aa1847/docs/implicit/args.md?plain=1#L17 for a well formatted example.